src/Entity/ShopBillingDataInterface.php

```diff
@@ -13,19 +13,35 @@
 
 namespace Sylius\RefundPlugin\Entity;
 
-interface ShopBillingDataInterface
+use Sylius\Component\Resource\Model\ResourceInterface;
+
+interface ShopBillingDataInterface extends ResourceInterface
 {
-    public function id(): ?int;
+    public function getId(): ?int;
+
+    public function setId(?int $id): void;
+
+    public function getCompany(): ?string;
+
+    public function setCompany(?string $company): void;
+
+    public function getTaxId(): ?string;
+
+    public function setTaxId(?string $taxId): void;
+
+    public function getCountryCode(): ?string;
+
+    public function setCountryCode(?string $countryCode): void;
 
-    public function company(): ?string;
+    public function getStreet(): ?string;
 
-    public function taxId(): ?string;
+    public function setStreet(?string $street): void;
 
-    public function countryCode(): ?string;
+    public function getCity(): ?string;
 
-    public function street(): ?string;
+    public function setCity(?string $city): void;
 
-    public function city(): ?string;
+    public function getPostcode(): ?string;
 
-    public function postcode(): ?string;
+    public function setPostcode(?string $postcode): void;
```

Comment:
As I see, you didn't remove these methods in `ShopBillingData` class, so why did you change them here? I would be for deprecating them instead of removing

Reply:
i got them back along with deprecation info